### ISSUE TITLE
Cutoff vs cellwidth

### ISSUE THREAD

**Issue by TSGut** [tjjarvinen/NeighborListTools.jl#1]
In the current structs the cells are unaware of their sidelengths, I believe with the idea being that CellList knows the cutoff and the cutoff would be identical to the cell width. But if we want to test decoupling those as suggested in some of the literature, we should probably add some sort of width knowledge to the cells themselves.

Actually maybe this is just a naming convention thing. Maybe cutoff should instead be called cellwidth or cellsidelength or something and then only at the level of the NeighborList do we actually keep around a cutoff which is then not necessarily equal to the cellwidth?

**Comment by tjjarvinen** [comment]
Yes, cellwidth is a better name. But we need to think what information is must have and what is optional...

**Comment by TSGut** [comment]
I was thinking particle number in a cell is implicit in the positions but whether or not storing it separately makes sense depends on how often this number is called. If it is called very often the constructing it over and over from the position list is less efficient than storing a single integer. So while this is optional we probably want to keep it

**Comment by tjjarvinen** [comment]
@TSGut I added reference data to the repo. You can look [here](https://github.com/tjjarvinen/NeighborListTools.jl/blob/main/scripts/example-data.jl) on how to use it.

With this you should be able to begin prototyping yourself.

**Comment by TSGut** [comment]
Awesome, I'll be taking a look!
